.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "pypy3"]
+
+    steps:
+      - name: Install libgnutls28-dev
+        run: |
+          sudo apt update -q
+          sudo apt install -q -y libgnutls28-dev libcurl4-gnutls-dev
+      
+      - uses: actions/checkout@v1
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install project dependencies
+        run: |
+          python -m pip install -U pip codecov tox tox-gh-actions
```

Comment:
Is `-U` for `--user` or for `--upgrade`?  Both are valid pip options.  Using `--upgrage` makes intentions clear to the reader.

Reply:
@cclauss `-U` is for upgrade! `--user` has no alias.

``` 
-U, --upgrade               Upgrade all specified packages to the newest available version. The handling of dependencies depends on the upgrade-strategy used.
```

Reply:
I know that but does the ___reader___ know that?  Why should they be forced to guess?  My point is that `--upgrade` is more self-documenting than `-U`.

Reply:
@cclauss I get your point. Can you open a PR to improve this?